### PR TITLE
DB-172518 Make scmdb supports SQLcl version 20 

### PR DESCRIPTION
**Pull request by MobyDrake**
sql20comp

### CODE REVIEW

File: src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java [hunk 1]
```diff
@@ -50,6 +56,19 @@ public SqlScriptExecutor() {
         executor.setStreamHandler(new PumpStreamHandler(new LogOutputStream() {
             @Override
             protected void processLine(String line, int logLevel) {
+                if (line.contains(INVALID_OBJECT_IN_SCHEMA)) {
```
Comment (Simplycissimys) [inline src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java:59]: This solution looks unstable, please review 'compile_invalids_wrapper_fail_on_error.sql' and provide more reliable one

**Review by Simplycissimys — CHANGES_REQUESTED** [review]

File: src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java (hunk 1 above)
Comment (MobyDrake) [reply src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java]: New way.
Found a system variable disabling result headers. Now output 20 equals 19.

File: src/main/resources/compile_invalids_wrapper_fail_on_error.sql [hunk 2]
```diff
@@ -104,12 +104,15 @@ begin
...
 
+set heading off
 select 'Invalid objects in [' || user || ']:'
 from dual;
+
 column object_type format a15
 column object_name format a30 wra
 column invalid_message format a15
 set tab off
+set heading off
```
Comment (Simplycissimys) [inline src/main/resources/compile_invalids_wrapper_fail_on_error.sql:115]: should be `set heading on` ?

File: src/main/resources/compile_invalids_wrapper_fail_on_error.sql [hunk 3]
```diff
@@ -104,12 +104,15 @@ begin
 end;
 /
 
+set heading off
```
Comment (Simplycissimys) [inline src/main/resources/compile_invalids_wrapper_fail_on_error.sql:107]: do the same in `compile_invalids_wrapper_not_fail_on_error.sql`

**Review by Simplycissimys — CHANGES_REQUESTED** [review]

File: src/main/resources/compile_invalids_wrapper_fail_on_error.sql (hunk 2 above)
Comment (MobyDrake) [reply src/main/resources/compile_invalids_wrapper_fail_on_error.sql:115]: Then the header will be duplicated for each object.
image

File: src/main/resources/compile_invalids_wrapper_fail_on_error.sql (hunk 3 above)
Comment (MobyDrake) [reply src/main/resources/compile_invalids_wrapper_fail_on_error.sql:107]: Done

**Review by Simplycissimys — APPROVED** [review]

**Pull request merged**